src/pages/similarity-indexes.html

```diff
@@ -43,47 +43,47 @@ <h3 class="mb-0"><span class="collapsible-heading" data-toggle="collapse" data-t
                         class="index-row"
 						ng-class="{'table-info text-secondary':index.name == selected.name}">
 						<td>
-                            <a class="lead index-name" ng-href="{{(index.status == 'BUILT' || index.status == 'REBUILDING' || index.status == 'OUTDATED') ? '#' : ''}}" ng-click="goToSimilarityIndex(index)">{{index.name}}</a>
+                            <a class="lead index-name" ng-href="{{(similarityIndexStatus.BUILT === index.status || similarityIndexStatus.REBUILDING === index.status || similarityIndexStatus.OUTDATED === index.status) ? '#' : ''}}" ng-click="goToSimilarityIndex(index)">{{index.name}}</a>
```

Comment:
I would consider transforming the loaded indexes data into some UI model where all these to be precalculated and part of the UI model so that this template to be simplified.

Reply:
Yes I will do it, but step by step. Next MR will do this.